src/components/common/CodeReview.css

```diff
@@ -0,0 +1,40 @@
+.line-number {
```

Comment:
Try to avoid using CSS. You can use CSS using `makeStyles` from Material UI. Check out how we use it in other parts of the app (or the Code App), for example.